services/aster-nix/src/fs/mod.rs

```diff
@@ -14,27 +14,47 @@ pub mod ramfs;
 pub mod rootfs;
 pub mod utils;
 
+use crate::fs::exfat::{ExfatFS, ExfatMountOptions};
 use crate::fs::{ext2::Ext2, fs_resolver::FsPath};
 use crate::prelude::*;
 use crate::thread::kernel_thread::KernelThreadExt;
+use alloc::format;
+use aster_block::BlockDevice;
 use aster_virtio::device::block::device::BlockDevice as VirtIoBlockDevice;
 use aster_virtio::device::block::DEVICE_NAME as VIRTIO_BLOCK_NAME;
 
+fn start_block_device(device_name: &str) -> Result<Arc<dyn BlockDevice>> {
+    if let Some(device) = aster_block::get_device(device_name) {
+        let cloned_device = device.clone();
+        let task_fn = move || {
+            info!("spawn the virt-io-block thread");
+            let virtio_block_device = cloned_device.downcast_ref::<VirtIoBlockDevice>().unwrap();
+            loop {
+                virtio_block_device.handle_requests();
+            }
+        };
+        crate::Thread::spawn_kernel_thread(crate::ThreadOptions::new(task_fn));
+        Ok(device)
+    } else {
+        return_errno_with_message!(Errno::ENOENT, "Device does not exist")
+    }
+}
+
 pub fn lazy_init() {
-    let block_device = aster_block::get_device(VIRTIO_BLOCK_NAME).unwrap();
-    let cloned_block_device = block_device.clone();
+    let ext2_device_name = format!("{}{}", VIRTIO_BLOCK_NAME, "6");
```

Comment:
I think using drive id is better

Reply:
Fixed, we use 'vexfat' and 'vext2' as the name of the two drives.